azkaban-exec-server/src/main/java/azkaban/execapp/FlowRampManager.java

```diff
@@ -321,7 +320,7 @@ synchronized void saveSettings() {
         .entrySet()
         .stream()
         .forEach(this::updateExecutedRampFlows);
-    rampDataModel.resetEndFlowCount();
+    rampDataModel.resetFlowCountsAfterSave();
```

Comment:
resetFlowCountAfterSave, the "s" is not needed.

Reply:
addressing